flixel/system/debug/Console.hx

```diff
@@ -112,7 +112,7 @@ class Console extends Window
 		_input.addEventListener(FocusEvent.FOCUS_OUT, onFocusLost);
 		_input.addEventListener(KeyboardEvent.KEY_DOWN, onKeyPress);
 		
-		#if (cpp || neko) // workaround for broken TextField focus on native
```

Comment:
By "just work" you mean keyboard listeners not being broken, rather than the console working (there's no input text field support in next yet is there?).